torchdata/datapipes/iter/load/README.md

```diff
@@ -1,37 +1,35 @@
 # S3 IO Datapipe Documentation
 
-## Installation
+## Build from Source
 
-Torchdata S3 IO datapipes depends on [aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp). The following is just a
-recommended way to installing aws-sdk-cpp, please refer to official documentation for detailed instructions.
+`ninja` is required to link PyThon implementation to C++ source code.
 
 ```bash
-git clone --recurse-submodules https://github.com/aws/aws-sdk-cpp
-cd aws-sdk-cpp/
-mkdir sdk-build
-cd sdk-build
-# need to add flag -DBUILD_SHARED_LIBS=OFF for static linking on Windows
-cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer"
-make
-make install # may need sudo
+conda install ninja
 ```
 
-`ninja` and `pybind11` are also required to link PyThon implementation to C++ source code.
+S3 IO datapipes are included when building with flag `BUILD_S3=1`. The following commands can build `torchdata` from
+source with S3 datapipes.
 
 ```bash
-conda install ninja pybind11
-```
-
-S3 IO datapipes are't included when building by default. To build S3 IO in `torchdata`, at the `/data` root folder, run
-the following commands.
-
-```bash
-export BUILD_S3=ON
 pip uninstall torchdata -y
+pip uninstall torchdata -y
+git clone https://github.com/pytorch/data.git
+cd data
 python setup.py clean
-python setup.py install
+BUILD_S3=1 python setup.py install
```

Comment:
nit: duplicate `uninstall`

Could you please also add this link https://github.com/pytorch/data/tree/main/torchdata/datapipes/iter/load#installation to https://github.com/pytorch/data/blob/main/README.md (in from source section) to let users know they can install with AWSSDK enabled.

Reply:
Updated.